src/descriptors/power_system_structs.json

```diff
@@ -94,36 +94,30 @@
       "supertype": "OperationalCost"
     },
     {
-      "struct_name": "MultiStartCost",
-      "docstring": "Data Structure Operational Cost Data which includes fixed, variable cost, multiple start up cost and stop costs.",
+      "struct_name": "MarketBidCost",
+      "docstring": "Data Structure Operational Cost to reflect market bids of energy and ancilliary services.\nCompatible with most US Market bidding mechanisms",
       "fields": [
         {
-          "name": "variable",
+          "name": "energy_bid",
           "comment": "variable cost",
           "null_value": "VariableCost((0.0, 0.0))",
           "data_type": "VariableCost"
         },
         {
-          "name": "no_load",
+          "name": "no_load_cost",
           "comment": "no load cost",
           "null_value": "0.0",
           "data_type": "Float64"
         },
         {
-          "name": "fixed",
-          "comment": "fixed cost",
-          "null_value": "0.0",
-          "data_type": "Float64"
+          "name": "start_up",
+          "comment": "start-up cost",
+          "null_value": "(hot = START_COST, int = START_COST, cold = START_COST)",
```

Comment:
could we also add a more substantial comment too describe what the intended usage is?